src/main/java/com/segment/analytics/android/integrations/nielsendcr/NielsenDCRIntegration.java

```diff
@@ -230,9 +261,25 @@ private JSONObject mapSpecialKeys(
       adContentMetadata.put("segC", segC);
     }
 
-    int contentAssetId = properties.getInt("contentAssetId", 0);
+    String assetIdPropertyName =
+        (settings.assetIdPropertyName != null) ? settings.assetIdPropertyName : "contentAssetId";
+    int contentAssetId = properties.getInt(assetIdPropertyName, 0);
     adContentMetadata.put("assetid", String.valueOf(contentAssetId));
 
+    String clientIdPropertyName =
+        (settings.clientIdPropertyName != null) ? settings.clientIdPropertyName : "clientId";
+    String clientId = properties.getString(clientIdPropertyName);
+    if (clientId != null && !clientId.isEmpty()) {
+      adContentMetadata.put("clientId", clientId);
```

Comment:
same here: Per the nielsen docs the key should be lowercase clientid in the metadata
https://engineeringportal.nielsen.com/docs/Digital_Measurement_Metadata?origin_team=T026HRLC7